Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -64,7 +65,18 @@ public final class GDSInformationViewController: BaseViewController, TitledViewC
             }
         }
     }
-    
+
+    /// Stack View: `UIStackView`. Any `UIView` which is on the `GDSInformationViewModelWithChildView` view model's `childView` property.
+    /// This will be added to the `stackView` below the existing `bodyLabel`
+    @IBOutlet private var stackView: UIStackView! {
+        didSet {
+            if let childView = (viewModel as? GDSInformationViewModelWithChildView)?.childView {
```

Comment:
I think this might be a little cleaner if we split these if lets 

```
if let viewModel = viewModel as? GDSInformationViewModelWithChildView, 
   let childView = viewModel.childView {

}
```